src/Accounts/Accounts/Az.Accounts.psd1

```diff
@@ -142,8 +142,10 @@ PrivateData = @{
         # IconUri = ''
 
         # ReleaseNotes of this module
-        ReleaseNotes = '* Add ''Register-AzModule'' command to support AutoRest generated cmdlets
-* Update examples for Connect-AzAccount'
+        ReleaseNotes = '* Azure automation update management change to support the following new features :
```

Comment:
@markcowl  To fix this I need to rerun the versioning tool since Account module doesn't have changes in this release and its version should have not been bumped up. So we decided to merge all updated directly to the release branch and then do the version update with the RC2. This PR will be closed.